sdk/include/reactos/mc/errcodes.mc

```diff
@@ -43001,7 +43121,7 @@ Severity=Warning
 Facility=WIN32
 SymbolicName=E_ACCESSDENIED
 Language=English
-WIN32 access denied error
+General access denied error
```

Comment:
Update all languages.

Reply:
I don't know what the correct translation on these systems is.

Reply:
You can revert to English in such cases.